### PR TITLE
Update python patch versions

### PR DESCRIPTION
**Pull request by Javier-Borreguero**
### Checklist

- [x] I have run the complete test suite to ensure all tests and linters pass.
- [x] I have thoroughly tested my code changes to ensure they work as expected, including adding additional tests for new functionality.
- [x] I have written clear and descriptive commit messages.
- [x] I have provided a detailed description of the changes in the pull request, including the problem it addresses, how it fixes the problem, and any relevant details about the implementation.
- [x] I have ensured that the code is well-documented and easy to understand.

### CODE REVIEW

**Review by JamieMagee — APPROVED** [review]

**Comment by Javier-Borreguero** [comment]
hello @JamieMagee , seems like we need an approval from any maintainer. I cannot add that team as reviewer, could you? Thanks!

**Comment by Javier-Borreguero** [comment]
@abdulapopoola could you merge, please?

**Comment by Javier-Borreguero** [comment]
Hey guys, can you provide some support here to merge this? @JamieMagee @abdulapopoola thank you so much

**Review by kbukum1 — APPROVED** [review]

**Pull request merged**
